src/lib/construct/api-to-eventbridge-target/main.ts

```diff
@@ -413,14 +413,13 @@ export class ApiToEventBridgeTarget extends CommonConstruct {
    */
   protected createApiDestinedIntegrationResponse() {
     if (!this.props.api.withResource) return
-    this.apiDestinedRestApi.integrationResponse = {
+    this.apiDestinedRestApi.integrationResponse = this.props.api.integrationResponse ?? {
```

Comment:
@hemalshah-gradientedge FYI... this should enable to just override the response template with cdk props only without having to override the method